lib/umwelt/client/clone.ex

```diff
@@ -0,0 +1,95 @@
+defmodule Umwelt.Client.Clone do
+  @moduledoc "Clone main process"
+
+  use GenServer
```

Comment:
```suggestion
  use GenServer, restart: :transient, shutdown: 10_000
```

Default for `:restart` is [`:permanent`](https://hexdocs.pm/elixir/GenServer.html#module-how-to-supervise) and I doubt restarting makes any sense here.

Reply:
Looks like this change fixed both tests.